server/app/middlewares/websocket_backend.rb

```diff
@@ -293,25 +290,20 @@ def handle_rpc_response(data)
   # @param [Faye::WebSocket::Event] ws
   # @param [Array] data
   def handle_rpc_request(ws, data)
-    client = client_for_ws(ws)
-    if client
-      self.rpc_queue << [ws, client[:grid_id].to_s, data]
+    if client = client_for_ws(ws)
+      RpcServer.handle_rpc_request(client[:grid_id].to_s, data) do |message|
```

Comment:
Does this improve performance (because it skips mailbox)?

Reply:
No, the `RpcServer.queue` is unchanged, it's just refactored so that the `WebsocketBackend` class doesn't need to know about it.